### ISSUE TITLE
fg push fg: job not found: push

### ISSUE THREAD

**Issue by okonet** [nvbn/thefuck#135]
I've updated to the latest version and still having issues on OS X 10.10.3:

```
» g push
fatal: The current branch feature/[branch name] has no upstream branch.
To push the current branch and set the remote as upstream, use

    git push --set-upstream origin feature/[branch name]

» fuck                                                                                       128 ↵
fg push
fg: job not found: push
```

**Comment by kimtree** [comment]
As I commented in the pull request above, "g push" command is a alias command so It is quite tricky to support to this as well.

**Comment by sullyj3** [comment]
Should have been fixed by fcc2a1a40a6e07a76cc653352c052e810df0546f.

**Comment by nvbn** [comment]
Support of aliases was added in 1.40 (or before).
